nss-day-cohort-41/tabloidmvc-the-lobster-rolls
# Bs category 1

# Description
Added functionality for Category table. Users can now view a Category Management list, in addition to the list view, the user can add, edit, and delete a category.
Fixes # 9,10,11,12
## Type of change
Please delete options that are not relevant.

# Testing Instructions
Pull down remote branch using Git requests in the console
*Adjustments made to Create SQL file to include IsDeleted Columns with default values of 0 to Post and Tags
*Adjusted seed post to have a new categoryId of 2

Run Sql Script to Create TabloidMVC database,
Run Sql Script to SEED TabloidMVC database
From Visual Studio -> Execute TabloidMVC
	
	Category Management:
       1. Click “Category Management” from Navbar
       2. View list of Categories, available to be used in Posts
>	
	Category Create:
        1. From Category Management View, Click “Add New Category” located the top of the view
        2. Add New Category by Name
        3. Click “Save” to complete Adding new Category
        4. Or Click “Back to List” to return to the Category List 
>
     Category Edit:
        1. From Category Management View, Click Edit on any Category item
        2. Update the category Name 
        3. Click “Save” to complete Edit
        4. Or Click “Cancel” to return to the Category List
>
     Category Delete:
        1. From Category Management View, Click Delete on any Category item
        2. Confirm the category Name 
        3. Click “Delete” to complete Delete
        4. Or Click “Cancel” to return to the Category List
        5. Once Category has been Deleted, any post with that category will now have a category name of "Other". 
        6. The "Other" Category can not be deleted



 
# Checklist:
